packages/template-retail-react-app/app/components/store-locator-modal/stores-list.jsx

```diff
@@ -15,78 +15,113 @@ import {
+                                    {store.address1}
+                                </Box>
+                                <Box fontSize="md" color="gray.600">
+                                    {store.city}, {store.stateCode ? store.stateCode : ''}{' '}
+                                    {store.postalCode}
+                                </Box>
+                                {store.distance !== undefined && (
+                                    <>
+                                        <br />
+                                        <Box fontSize="md" color="gray.600">
+                                            {store.distance} {store.distanceUnit}{' '}
+                                            {intl.formatMessage({
+                                                id: 'store_locator.description.away',
+                                                defaultMessage: 'away'
+                                            })}
+                                        </Box>
+                                    </>
+                                )}
+                                {store.phone && (
+                                    <>
+                                        <br />
+                                        <Box fontSize="md" color="gray.600">
+                                            {intl.formatMessage({
+                                                id: 'store_locator.description.phone',
+                                                defaultMessage: 'Phone:'
+                                            })}{' '}
+                                            {store.phone}
+                                        </Box>
+                                    </>
+                                )}
+                                {store.storeHours && (
+                                    <>
+                                        {' '}
+                                        <AccordionButton
+                                            color="blue.700"
+                                            style={{marginTop: '10px', paddingBottom: '0px'}}
+                                        >
+                                            <Box fontSize="lg">
+                                                {intl.formatMessage({
+                                                    id: 'store_locator.action.viewMore',
+                                                    defaultMessage: 'View More'
+                                                })}
+                                            </Box>
+                                            <AccordionIcon />
+                                        </AccordionButton>
+                                        <AccordionPanel mb={6} mt={4}>
+                                            <div
+                                                dangerouslySetInnerHTML={{
+                                                    __html: store?.storeHours
```

Comment:
Why `dangerouslySetInnerHTML` here? Can't we just interpolate the value in the place where we declare vars via `useState` and similar?

Reply:
The `/store-search` API actually returns the store hours as html 
<img width="1787" alt="Screenshot 2025-01-17 at 2 15 51 PM" src="https://github.com/user-attachments/assets/162f78f6-a08e-4f3a-8f85-e419278959c0" />

Reply:
Oh wow, SUPER weird, but ok 🚢